.github/workflows/hyperion_java-client_publish.yml

```
@@ -0,0 +1,124 @@
+            hyperion/java-client/build/reports/
+          retention-days: 7
+
+  publish-snapshot:
+    name: Publish Snapshot to GitHub Packages
+    needs: validate
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)